src/net/gosha/atproto/firehose_analysis.clj

```diff
@@ -0,0 +1,247 @@
+(ns net.gosha.atproto.firehose-analysis
```

Comment:
I'd move this to a separate top-level "examples" dir, since this kind of analysis is useful but isn't part of the Bluesky Client proper.